docker_config/compose_files/docker-compose.yml

```diff
@@ -67,6 +67,8 @@ x-codalab-env: &codalab-env
   - CODALAB_ALWAYS_USE_AZURE_BLOB_BETA=${CODALAB_ALWAYS_USE_AZURE_BLOB_BETA}
   - CODALAB_SENTRY_INGEST_URL=${CODALAB_SENTRY_INGEST_URL}
   - CODALAB_SENTRY_ENVIRONMENT=${CODALAB_SENTRY_ENVIRONMENT}
+  - CODALAB_RECAPTCHA_SECRET_KEY=${CODALAB_RECAPTCHA_SECRET_KEY}
+  - REACT_APP_CODALAB_RECAPTCHA_SITE_KEY=${CODALAB_RECAPTCHA_SITE_KEY}
```

Comment:
Without this line users cannot supply their own site_key in development mode i think? Since in dev mode the react app actually reads the docker's env vars.

Reply:
I'm pretty sure that the env var will still be passed to the react app (due to the presence of .env file) in dev mode. Maybe you could try and see if dev mode still gets the env variable when taking this line out?

Reply:
I think the edge case is when user only use `-d` instead of `-bd`, then the .env is not outputted thus the key cannot be read.

Reply:
Yeah I think this is probably the expected behavior I went ahead and applied the suggestion.